warehouse/forklift/legacy.py

```diff
@@ -571,6 +571,22 @@ def _is_valid_dist_file(filename, filetype):
     # allow it.
     return True
 
+def _is_duplicate_file(db_session, filename, hashes):
+    """ Check to see if file already exists, and if it's content matches
+    """
+
+    file_ = (
+        db_session.query(File)
+                  .filter(File.filename == filename)
+                  .first()
+    )
+
+    if file_:
```

Comment:
I'd prefer to explicitly check for ``None`` here, it shouldn't matter but just in case I think it makes sense to make this ``if file_ is not None:``. Doesn't hurt at least.

Reply:
A+, makes sense to be safe.